### DISCUSSION TITLE
I have a question bro

### DISCUSSION THREAD

**Discussion by XahidEx** [theodorusclarence/theodorusclarence.com#270] [Ideas]
I'm a newbie. So you might get annoyed because of the question. The question is how do I add another language? Like I'm from Bangladesh. I want to use Bengali. Would you please kindly guide me. 🙈

**Comment by theodorusclarence** [comment]
I've never built a website with multi-language before, but you can use i18n library like https://github.com/isaachinman/next-i18next

**Reply by XahidEx** [reply]
But, How did you use the Indonesian language? Did you put the translation on your own? I just want to use Bengali instead of using the Indonesian language. Because my blog will be Bengali.

**Reply by theodorusclarence** [reply]
You basically write 2 blog post with both language, you can check it here https://github.com/theodorusclarence/theodorusclarence.com/blob/main/src/contents/blog/id-btb-flexbox-grid.mdx and https://github.com/theodorusclarence/theodorusclarence.com/blob/main/src/contents/blog/btb-flexbox-grid.mdx
